### ISSUE TITLE
dero wallet import.....

### ISSUE THREAD

**Issue by Romeo01234** [deroproject/derohe#110]
I am trying to import my old dero wallet to the new derohe wallet and I show zero coins.

How do i transfer/retrieve the coins into the new derohe wallet?

Thanks.

**Comment by Fka27ADA-mod** [comment]
@Romeo01234 Have you gotten any help and able to resolved your issues?

**Comment by RamonDev336** [comment]
@Romeo01234 Talk to a Dero Support on the [Live support page](https://explore-dero.online) so that your issue can be looked into.

**Comment by Slixe** [comment]
Not at all, its a scam. ANN is coming in coming weeks, join the real discord, never share any private key/info/seed to anyone.

**Comment by Romeo01234** [comment]
What is a scam???

**Comment by secretnamebasis** [comment]
> I am trying to import my old dero wallet to the new derohe wallet and I show zero coins.
> 
> How do i transfer/retrieve the coins into the new derohe wallet?
> 
> Thanks.

Sounds like you missed the "swap" for balance transfers. 
Please go to discord and await for further instructions in the #announcement channel. 
https://discord.gg/H95TJDp

**Comment by Romeo01234** [comment]
When will the announcement take place?

I see the last entry is in June 2023.
